### PR TITLE
layer: Updated synchronization preset

### PR DESCRIPTION
**Pull request by christophe-lunarg**
This PR changes the Synchronization Preset from:
image

To:
image

### CODE REVIEW

**Comment by jzulauf-lunarg** [comment]
I think we should leave the thread safety enabled.  Will update my best practices recommendations to match the preset with that.

**Review by jzulauf-lunarg — CHANGES_REQUESTED** [review]
Let's leave thread safety enabled.  I've changed all  my materials to match unique_handles and thread safety on.

File: layers/VkLayer_khronos_validation.json.in [hunk 1]
```diff
@@ -324,7 +324,7 @@
                         },
                         {
                             "key": "thread_safety",
-                            "value": true
+                            "value": false
```
[the remark this answers is not in the document]
Comment (christophe-lunarg) [reply layers/VkLayer_khronos_validation.json.in]: Fixed

**Comment by christophe-lunarg** [comment]
> Let's leave thread safety enabled. I've changed all my materials to match unique_handles and thread safety on.

I explicitly added sync_queue_submit to true even if it's not required as it's the default value for confort.

**Review by jzulauf-lunarg — APPROVED** [review]
LGTM

**Pull request merged**